Jenkinsfile

```diff
@@ -25,7 +25,7 @@ def mailBuildResults(String label, additionalEmails='') {
 
 def runTests(String testPath) {
     sh "./runTests.py -j${env.PARALLEL} ${testPath} --make-only"
-    try { sh "./runTests.py -j${env.PARALLEL} ${testPath}"}
+    try { sh "./runTests.py -j${env.PARALLEL} ${testPath}" }
```

Comment:
I'm so sorry, that was a joke, haha. I should have added another emoji to explain that, lol.

Reply:
lol it's fine I knew it was a joke

Reply:
Hence my commit message in 514c39f